csharp/CMakeLists.txt

```diff
@@ -48,61 +40,36 @@ endif()
 
 # the main OpenStudio.dll generated
 
-if (MSVC)
-  # TODO: Remove
-  # set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} /FORCE:MULTIPLE")
-
-  if(NINJA)
-    set(OPENSTUDIO_CSHARP_DLL "${CSHARP_LIBRARY_OUTPUT_DIRECTORY}/OpenStudio.dll")
-    set(CONFIG_ARGS "/p:Configuration=${CMAKE_BUILD_TYPE}")
-  else()
-    # **Regular case, pure MSVC**
-    set(OPENSTUDIO_CSHARP_DLL "${CSHARP_LIBRARY_OUTPUT_DIRECTORY}/$(ConfigurationName)/OpenStudio.dll")
-    set(CONFIG_ARGS "/p:Configuration=$(ConfigurationName)")
-  endif()
+# Rely on dotnet Core (open source)
+find_program(DOTNET dotnet)
+if(NOT DOTNET)
+  message(FATAL_ERROR "C# generation outside of MSVC requires .NET Core (dotnet)")
+endif()
```

Comment:
This is the main change that makes me a bit nervous. When I added dotnet support, I kept the MSVC in place to make sure it wouldn't break existing workflows. I do not know much about C# though, so that was just being cautious. Can anyone tell me if there are/which are the differences between building via MSVC .NET Framework versus .NET Core?

Reply:
The new dotnet can also be used to compile the old .Net framework project. So it is the same.

Reply:
@MingboPeng  is building it for .net standard 2.0 which should be compatible with .net framework that most people are using